User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+<!--- Explanation of what Trust Levels, how they are calculated, and how to earn then -->
+
+# Trust Level Overview
+
+Because Codidact is community run, it doesn't have owners who dictate behavior. Instead, users who consistently make solid contributions through posting and other activity on a community gradually obtain more privileges and access to certain moderation tools based on a calculated "Trust Level." You can see current Trust Level on any user's profile. 
```

Comment:
Punctuation nits: "community-run" (hyphen because adjective), and semicolon after "run" because these are two top-level clauses.